openlibrary/templates/site/banner.html

```diff
@@ -0,0 +1,22 @@
+$def with (announcement, storage_key)
+
+$ hidden = False
+$if ctx.user and ctx.user.preferences().get('hidden-banner') == storage_key:
+    $ hidden = True
+
+$def hideBanner():
+    $ctx.user.save_preferences({'hidden-banner': storage_key})
+
+$# Paste next line into the span element to show the blue banner
+$# <strong>Title</strong> $:announcement &nbsp; <a href="#" data-ol-link-track="AnnouncementsBar|BlogWidget" class="btn primary">Learn More</a>
```

Comment:
Is this comment helpful?

Reply:
Yes @jimchamp, I think it will make it easier for the next developer who would like to include the blue banner.